src/app/core/shared/value-list-browse-definition.model.ts

```diff
@@ -31,6 +31,6 @@ export class ValueListBrowseDefinition extends NonHierarchicalBrowseDefinition {
   };
 
   getRenderType(): string {
-    return this.dataType;
+    return this.type.value;
```

Comment:
I encountered a few similar issues today while resolving merge conflicts on #2865, which included upgrading TSESLint and introducing a new default rule for enum consistency
- There were a few cases these `getRenderTypes` calls being compared with a ResourceType, and I was considering replacing them with `type` instead
- If we change this browse definition we should probably do the same for the other ones
- If we use `this.type` here, do we still need the method at all?

Reply:
https://github.com/DSpace/dspace-angular/blob/fd000cdfc4b4b53074e1fe8b1b6b9a10f383305e/src/app/core/shared/browse-definition.model.ts#L15

The abstract inheritance requires it of implementations. I don't think I would make that many changes to resolve a bug. The component and class structure might require revisiting and that would be a scope creep within this patch. Additionally, as the unit test required breaking apart and it went under the coverage of e2e tests, I would think e2e tests are required to handle an instance of significant refactoring. Then again, I have seen larger changes to fix a bug before. 🥁

Reply:
Perhaps e2e test could be in the forward port to the next release.

Reply:
Good point about the scope creep, should probably be it's own issue/PR 👍